src/pages/about.tsx

```diff
@@ -6,7 +6,7 @@ import { Layout } from "../components/Layout";
 const About = () => {
   return (
     <Layout>
-      <Container>
+      <Container style={{ paddingTop: "50px" }}>
```

Comment:
Been wondering if we should add a [`<Spacer />`](https://www.joshwcomeau.com/react/modern-spacer-gif/) component for stuff like this, because I've been needing to do stuff like this too